lib/travis/build/addons/rethinkdb.rb

```diff
@@ -0,0 +1,39 @@
+require 'shellwords'
+require 'travis/build/addons/base'
+
+module Travis
+  module Build
+    class Addons
+      class Rethinkdb < Base
+        SUPER_USER_SAFE = true
+
+        RETHINKDB_GPG_KEY = '0x3A8F2399'
+
+        def after_prepare
+          sh.fold 'rethinkdb' do
+            sh.echo "Installing RethinkDB version #{rethinkdb_version}", ansi: :yellow
+            sh.cmd "service rethinkdb stop", sudo: true
```

Comment:
We have not been very vigilant about generating multi-platform scripts, but I think we should bail out if this addon runs on OS X.